src/Client/Stripe.php

```diff
@@ -80,8 +75,4 @@ public function createSetupIntent(StripeCustomerId $stripeCustomerId): SetupInte
     public function retrievePaymentMethod(StripeCustomerId $customerId, StripePaymentMethodId $methodId): PaymentMethod;
 
     public function detatchPaymentMethod(StripePaymentMethodId $paymentMethodId): void;
-
-    public function searchCustomers(array $searchParams): SearchResult|Collection;
-
-    public function listAllPaymentMethodsForCustomer(StripeCustomerId $id, array $params): Collection;
```

Comment:
I might not have deleted these if Stripe hadn't made me, but I think there is value in keeping only the functions we actually use here so we can somewhat at a glance which Stripe features matchbot uses.